apps/pl/scss/_scss2json.scss

```diff
@@ -29,13 +29,15 @@ $spacing-demo: $spacing;
 //@import '../_patterns/01-atoms/button/_button.scss';
 
 // Import the map of available custom font icons.
-@import '../../../source/_patterns/01-atoms/icon/scss/_icons-generated';
+@import '../../../source/_patterns/01-atoms/svgicon/scss/_icons-generated';
 
-$icons-demo: (
-  prefix: 'icon--',
-  fontname: 'iconfont',
-  icons: map-keys(map_get($__iconfont__data, 'iconfont')),
-);
+$svgicons: map-keys($sprites);
```

Comment:
Give us a comment here on what the format of this looks like.

Reply:
Added in 4c7aedc